docs/cloud/azureai/runtime-change-log.md

```diff
@@ -14,6 +14,14 @@ You can check the runtime image version from the flow execution log:
 
 ## Change log
 Default runtime image is continuously updated, and here we record the new features and fixed bugs of each image version.
+### 20240205.v2
+
+#### New features
+NA
+
+#### Bugs fixed
+- Fix the bug that azureml serving get UAI(user assigned identity) token will fail.
```

Comment:
How about rephrasing into "Fix the bug that deployed promptflow endpoint fails to get user assigned identity token."

Reply:
Fixed. Thanks